pyuvdata/uvdata.py

```diff
@@ -474,7 +474,7 @@ def set_telescope_params(self, overwrite=False):
             raise ValueError('Telescope {telescope_name} is not in '
                              'known_telescopes.'.format(telescope_name=self.telescope_name))
 
-    def baseline_to_antnums(self, baseline):
+    def baseline_to_antnums(self, baseline, attempt256=False):
```

Comment:
one more nitpick: this method doesn't need the `attempt256` keyword.

Reply:
Ah sorry, thought I'd gotten them all. Thanks for reviewing!